src/links.nim

```diff
@@ -1,21 +1,42 @@
 import options
+import sets
+import tables
 
-import bitable
 import common
 import rect
 import selection
 import utils
 
-export bitable
-
+const DefaultInitialSize = 64
```

Comment:
This is not performance sensitive code, so let's ditch the initial size setting.